TACC/tup-ui
# feat: TUP-667 add instagram to social media

## Overview

- Add "Instagram" to social media platforms.
- Add social media platforms setting in repo.[^1]

## Related

- [TUP-667](https://tacc-main.atlassian.net/browse/TUP-667)

## Changes

- **added** `TACC_SOCIAL_SHARE_PLATFORMS` to settings
- **added** `instagram` to social media platforms
- **changed** order of social media platforms

## Testing

> [!NOTE]
> Not available to test on https://dev.tup.tacc.utexas.edu/ until `TACC_SOCIAL_SHARE_PLATFORMS` setting is removed from server.

1. Open a news article.
2. See:
    1. LinkedIn
    2. Instagram
    3. Facebook
    4. Email
3. Verify clicking Instagram icon opens up option for user to share on Instagram.

## UI

...

[^1]: These have been on server directly, to fix emergency bug.
